### PR TITLE
update text for (i)'s

### PR DESCRIPTION
**Pull request by tahmeedshah**
update (i)'s text for BCH and ABSCH.

### CODE REVIEW

File: app/views/register/dashboard.html [hunk 1]
```diff
@@ -103,35 +103,36 @@ <h4 style="margin-top:0; margin-bottom:0; padding:0 0 5px 0;border-bottom:1px so
...
-                                        data-container="body"></span>
-                                </span>
-
+                                    Registered User 
+                                    <span ng-if="isBch" href="#" class="helpinfoInline fa fa-info-circle color-litegrey" data-toggle="tooltip" data-html="true" data-container="body" 
+                                        data-title="Regular users can create draft reference records and request their publication from the CBD Secretariat.">
+                                    </span><span ng-if="isAbs" href="#" class="helpinfoInline fa fa-info-circle color-litegrey" data-toggle="tooltip" data-html="true" data-container="body" 
+                                        data-title="Regular users can create draft reference records and request their publication from the CBD Secretariat.">
+                                    </span></span>
+
```
Comment (blaisefonseca) [inline app/views/register/dashboard.html:135]: if the message is same no need to have separate messages for abs and bch

File: app/views/register/dashboard.html [hunk 2]
```diff
@@ -353,9 +354,9 @@ <h4 style="padding:0;margin:0;">
                         <div class="row" style="padding:0;margin:0 1px 0 1px; border-bottom:1px solid #ddd; ">
                             <span class="color-orange"> Reference records
                                 <span href="#" class="helpinfoInline fa fa-info-circle color-litegrey" data-toggle="tooltip" data-html="true" data-container="body" ng-if="isAbs"
-                                    data-title="Reference records contain information relevant to the Protocol's implemenation. This information can be made available by any registered user (including representatives of Governments, indigenous and local communities, academia, nongovernmental organizations, research institutions, business representatives, etc.). All reference records are validated by the Secretariat prior to being made publically available." ></span>
+                                    data-title="Reference records include a number of ABS-related resources and information that can be submitted by any registered user and are validated by the Secretariat prior to their publication." ></span>
```
Comment (blaisefonseca) [inline app/views/register/dashboard.html:357]: update the national and reference records text in search page as well

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
I have put two comments

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
`app/app-data/realm-configurations-bch.json` `app/app-data/realm-configurations-absch.json` these two files need to be under individual protocol folders

should be
`app/app-data/bch/realm-configuration.json`
`app/app-data/abs/realm-configuration.json`

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
`Contact` is missing description

**Review by blaisefonseca — APPROVED** [review]

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**